backend/plugins/authorization.js

```diff
@@ -1,17 +1,17 @@
 import {contains} from 'ramda';
 import {getSetting} from '../settings.js';
 import {getAccessTokenCookieOptions} from '../constants.js';
-import {generateAndSaveAccessToken, homeUrl} from '../utils/authUtils';
+import {generateAndSaveAccessToken} from '../utils/authUtils';
 import Logger from '../logger';
 import fetch from 'node-fetch';
 
 const ESCAPED_ROUTES = [
-  new RegExp(`^${homeUrl()}/$`),
-  new RegExp(`^${homeUrl()}/ping$`),
-  new RegExp(`^${homeUrl()}/login$`),
-  new RegExp(`^${homeUrl()}/static/`),
-  new RegExp(`^${homeUrl()}/oauth2$`),
-  new RegExp(`^${homeUrl()}/oauth2/callback$`)
+  new RegExp('^/$'),
```

Comment:
Could you add a comment explaining why we mustn't prefix the RegExp with `${homeUrl}`, please?

Reply:
Otherwise, we'll make this mistake again.

Reply:
Added.. 💃 💃 ?